github/libp2p.yml

```diff
@@ -3381,6 +3451,13 @@ repositories:
     default_branch: main
     delete_branch_on_merge: false
     description: Connect a browser to another browser
+    files:
+      .github/pull_request_template.md:
+        content: .github/js_pull_request_template.md
+      .github/workflows/semantic-pull-request.yml:
+        content: .github/workflows/semantic-pull-request.yml
+      .github/workflows/stale.yml:
+        content: .github/workflows/stale.yml
```

Comment:
these changes were not added by me.. I created PR through github.. I can remove if needed